test/app.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+print(
```

Comment:
Is it correct that this test only prints?

Reply:
I've since removed this file.